label_studio_ml/examples/interactive_segmentation/requirements.txt

```diff
@@ -0,0 +1,4 @@
+-e git+https://github.com/heartexlabs/label-studio-segmentation#egg=label-studio-segmentation
+albumentations>0.5
+easydict
+tensorboard
```

Comment:
it's better to fix versions of these dependencies

Reply:
also `fatal: repository 'https://github.com/heartexlabs/label-studio-segmentation/' not found` maybe it is private